From 57dfeea75bc5300d5ac75bbd4c29e754aadf8a00 Mon Sep 17 00:00:00 2001
From: Samuel Gaist <samuel.gaist@edeltech.ch>
Date: Sun, 2 Oct 2016 21:19:39 +0200
Subject: [PATCH 1/1] Fixed build with MaxOSX10.12 SDK

This patch adds missing includes that allows to build the module with
the 10.12 SDK and Xcode 8.

Change-Id: Ieab48f6a0582b916ceecbbb9a01a4169d6ba53f5
---
 src/bluetooth/osx/osxbtcentralmanager_p.h | 8 ++++++++
 src/bluetooth/osx/osxbtledeviceinquiry.mm | 5 +++++
 src/bluetooth/osx/osxbtutility.mm         | 3 +++
 3 files changed, 16 insertions(+)

Last 2 merged in 5.15

diff --git a/src/bluetooth/osx/osxbtcentralmanager_p.h b/src/bluetooth/osx/osxbtcentralmanager_p.h
index e64e5ba..81c5e8d 100644
--- a/src/bluetooth/osx/osxbtcentralmanager_p.h
+++ b/src/bluetooth/osx/osxbtcentralmanager_p.h
@@ -65,6 +65,14 @@
 
 #include <Foundation/Foundation.h>
 
+#if QT_MAC_PLATFORM_SDK_EQUAL_OR_ABOVE(__MAC_10_12, __IPHONE_NA)
+#include <CoreBluetooth/CBService.h>
+#include <CoreBluetooth/CBCharacteristic.h>
+#include <CoreBluetooth/CBDescriptor.h>
+#include <CoreBluetooth/CBCentralManager.h>
+#include <CoreBluetooth/CBPeripheral.h>
+#endif
+
 @class QT_MANGLE_NAMESPACE(OSXBTCentralManager);
 
 QT_BEGIN_NAMESPACE
